### PR TITLE
[SWTCH-946] Fix  notification counter

### PR DESCRIPTION
**Pull request by kim-energyweb**
This PR is created by the request of Vadym. He can't login to his GitHub account.

### CODE REVIEW

**Review by jrhender — APPROVED** [review]
Looks good to me. @DawidGil-jlabs If you've got time to take a look as well, that would be great 🙇

File: src/app/layout/header/header.component.html [hunk 1]
```diff
@@ -79,8 +79,25 @@ <h4 class="mb-0 f-16 color-white">Tasks</h4>
                                             <mat-icon svgIcon="new-claim-icon"></mat-icon>
                                         </div>
                                         <div
-                                            class="media-body d-flex flex-row justify-content-between align-items-center">
```
Comment (DawidGil-jlabs) [inline src/app/layout/header/header.component.html:82]: Why so big spaces? Maybe we should stick with one way, I mean `<div class="">...</div>` without additional enters/tabs/spaces. What do you think?

File: src/app/layout/header/header.component.ts [hunk 2]
```diff
@@ -301,13 +298,24 @@ export class HeaderComponent implements OnInit, OnDestroy {
...
+            this._initZeroCheckerForPendingSyncCount();
+        } catch (e) {
             throw e;
         }
     }
 
-    private async _initAssetsOfferedToMeSyncCount(){
+    private _initZeroCheckerForPendingSyncCount(): void {
+        this.notifService.pendingSyncCount$
+            .pipe(takeUntil(this._subscription$))
```
Comment (DawidGil-jlabs) [inline src/app/layout/header/header.component.ts:309]: You can add here filter operator, i mean `filter(count => count === 0)`. That way subscribe doesn't need if statement

**Review by DawidGil-jlabs — APPROVED** [review]

File: src/app/layout/header/header.component.html [hunk 3]
```diff
@@ -79,8 +79,25 @@ <h4 class="mb-0 f-16 color-white">Tasks</h4>
...
+                                                class="media-body d-flex flex-row justify-content-between align-items-center">
```
Comment (DawidGil-jlabs) [inline src/app/layout/header/header.component.html:82]: Why so big spaces? Maybe we should stick with one way, I mean <div class="">...</div> without additional enters/tabs/spaces. What do you think?

**Comment by DawidGil-jlabs** [comment]
Yeah, for me it also looks good, that's why i approved it. Just added comments to start discussion :) i think we need to enable tslint and in feature eslint (tslint is deprecated in favor of eslint).

**Review by DawidGil-jlabs — APPROVED** [review]

**Review by DawidGil-jlabs — APPROVED** [review]

**Pull request merged**